src/migrations/Migration20230724194333.ts

```diff
@@ -0,0 +1,33 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20230724194333 extends Migration {
+  public async up(): Promise<void> {
+    if (process.env.CHAIN_ID == '1') {
```

Comment:
Very dangerous thing here to use process env in migrations and influence DB data based on ENV variables